website/sidebars.js

```diff
@@ -45,6 +45,7 @@ const sidebars = {
         'guide/android-dev-env',
         'guide/proguard-configuration',
         'guide/cucumber-js-integration',
+        'guide/testing-webviews',
```

Comment:
I'd definitely lift this up to at least after `guide/genymotion-saas`

Reply:
done